test/k8sT/Policies.go

```diff
@@ -1075,6 +1104,8 @@ var _ = Describe("K8sPolicyTest", func() {
 			// avoid incomplete teardown if any step fails.
 			_ = kubectl.Delete(demoYAML)
 			ExpectAllPodsTerminated(kubectl)
+			kubectl.DeleteHubbleClientPods(helpers.CiliumNamespace)
+			kubectl.DeleteHubbleRelay(helpers.CiliumNamespace)
```

Comment:
hubble-relay does not need to be removed, since you are not deploying it

Reply:
fair.